react/tic-tac-toe/src/app/index.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
+import { ConnectedRouter } from 'connected-react-router';
+import { Switch, BrowserRouter } from 'react-router-dom';
+
+import { history } from '../redux/store';
+import { ROUTES } from '../constants/gameConstants';
 
 import Game from './screens/Game';
+import Login from './screens/Login';
+import AuthRoute from './components/AuthRoute';
 
-import '../scss/application.scss';
 
 function App() {
-  return <Game />;
+  return (
+    <ConnectedRouter history={history}>
+      <BrowserRouter>
+        <Switch>
+          <AuthRoute exact path={ROUTES.LOGIN} component={Login} redirectPath={ROUTES.GAME} loginComponent />
```

Comment:
I'd delete this route and add a default route in case the user try to gets into an invalid route. Like so: 

```
<AuthRoute path={ROUTES.DEFAULT_ROUTE} component={Game} />
```

Finally, in your `AuthRoute/index.js` component you should write the following:

```
function AuthRoute({ component: Component, redirectPath, loggedIn, ...props }) {
  return loggedIn ? <Route {...props} component={Component} /> : <Redirect to={redirectPath} />
}

AuthRoute.defaultProps = {
  redirectPath: ROUTES.LOGIN
};
```